loop/storage/redis_migration.js

```diff
@@ -18,7 +18,8 @@ var SUPPORTED_OPERATIONS = [
 
 var MULTI_OPERATIONS = [
   'pttl', 'ttl', 'set', 'setex', 'psetex', 'sadd', 'srem', 'pexpire',
-  'expire', 'incr', 'decr', 'hmset', 'hset', 'hsetnx', 'hdel', 'del'
+  'expire', 'incr', 'decr', 'hmset', 'hset', 'hsetnx', 'hdel', 'del',
```

Comment:
Since we're adding a few multi operations, and changing how multi works here, a test would be appreciated to prove it's doing the right thing.